store.go

```diff
@@ -127,10 +127,17 @@ func (t *Table) ProcessFile(ctx context.Context, blockDir string, lastBlockTimes
 		return err
 	}
 
+	return t.filterRowGroups(ctx, buf, filter, rowGroups)
+}
+
+func (t *Table) filterRowGroups(ctx context.Context, buf *dynparquet.SerializedBuffer, filter TrueNegativeFilter, rowGroups chan<- dynparquet.DynamicRowGroup) error {
+	_, span := t.tracer.Start(ctx, "Table/IterateBucketBlocks/Iter/ProcessFile/filterRowGroups")
```

Comment:
Don't have hard feelings about this, but I think I'd go with `Table/filterRowGroups`  the call stack is after all visible on the tracing UI too.

Reply:
Oh good! I didn't know what the standard was for naming traces, I do like shorter myself